### ISSUE TITLE
Write ACRT issues in plain language

### ISSUE THREAD

**Issue by cthomasONRR** [DOI-ONRR/onrr.gov-site#3232]
Let's make a doc to more clearly explain each ACRT issue and where in ANDI to test.

**Comment by cthomasONRR** [comment]
Spreadsheet [here](https://doimspp.sharepoint.com/:x:/r/sites/NRRD/Shared%20Documents/General/Accessibility/2024_ACRT_Instructions.xlsx?d=w8b13bcaff0964982a0c8c0404b8d05e9&csf=1&web=1&e=WehZUs)

**Comment by cthomasONRR** [comment]
I did a brain dump of what I remember from our meeting yesterday. @lpgoldstein If you have time today before your leave, please feel free to add to the spreadsheet!

**Comment by eelzi-ONRR** [comment]
The WCAG site also explains each test name in simpler terms: https://www.w3.org/TR/WCAG22/

The issue descriptions come verbatim from Trusted Tester  since ACRT is intended to be used with TT. so you can also read more about them in the Trusted Tester folder in Teams.

**Comment by AlexandracmONRR** [comment]
All of my notes that I had were reflected in the document you created. Thank you @cthomasONRR

**Comment by cthomasONRR** [comment]
Closing this and will create a new issue for Lindsay next sprint if she wants to review/contribute.
